src/User/RegistrationForm.jsx

```diff
@@ -83,6 +99,15 @@ const RegistrationForm = () => {
             required
             maxLength="100"
           />
+          <label>Manager: </label>
+          <select onChange={handleChange} name="managerId" defaultValue="">
+            <option value="">---- select ----</option>
+            {managers.map(manager => (
```

Comment:
Also this should probably be users.map, because they're not managers at this point - it's just a list of users that you select to choose your manager from

Reply:
yeah I changed it to managers and manager because we already use user on that page, I'll find a better way to name them when this PR reopens.